geometry_msgs/PoseStamped pose_target
---
float64[] base_goal
float64[] configuration_goal
float64[] distance_to_goal
